command_mode.c

```diff
@@ -727,7 +732,12 @@ static void cmd_showbind(char *arg)
 	if (*key == 0)
 		goto err;
 
-	/* FIXME: remove spaces at end */
+	/* remove trailing spaces from key */
+	char *key_end = key + strlen(key);
+	while (isspace(*(key_end-1))) {
+		key_end--;
+	}
+	*key_end = 0;
```

Comment:
It accepts only a single byte which is not enough to determine if something is a space. E.g. "a ̈b" contains ' '.